mne/report.py

```diff
@@ -645,6 +665,8 @@ class Report(object):
     ----------
     info_fname : str
         Name of the file containing the info dictionary.
+    cov_fname : str
+        Name of the file containing the noise covariance.
```

Comment:
This has to go at the end (before `verbose`) to avoid breaking people's code. `Report` has been in mne-python for a while now...